### DISCUSSION TITLE
Conclusions of the developers meeting

### DISCUSSION THREAD

**Discussion by jfigui** [openradar/erad2022#72] [General]
Hello all,

Thanks a lot for the very fruitful discussion we had the Saturday before ERAD2022 in Locarno-Monti. And thanks @wolfidan  for hosting!

I list here my conclusions from the meeting. Since the discussion spun in all directions perhaps I forgot something. Please feel free to complete it:

- Publication of an update paper describing the current state of the open source software
- New format for the open radar science web page
  - https://openradarscience.org/ @mgrover and @kmuehlbauer 
  - https://github.com/openradar/openradar.github.io/issues/6
  - https://github.com/openradar/openradar.github.io/issues/8
- Opening up a discussion thread for general radar questions in discourse
  - @kmuehlbauer - No response from the discourse team so far.
  - More info: https://github.com/openradar/erad2022/discussions/68
- Development of a stand-alone software package in python to read any moment radar data format and put it in an object based on xarray
  - https://github.com/openradar/xradar @mgrover1 and @kmuehlbauer
  - https://github.com/openradar/xradar/pull/13
  
- Creation of a data repository that can be used to objectively test algorithms

It would be good if we attach names and dates to those tasks.

Cheers,

Jordi

**Comment by kmuehlbauer** [comment]
Thanks @jfigui, I've filled in some items.
